### ISSUE TITLE
[FEATURE] Custom cursor instead of the default cursor

### ISSUE THREAD

**Issue by sreevyshnavi02** [prakhartiwari0/Arito#77]
Since the website is primarily meant for kids to learn basic math, a custom arrow would add more impact. Since the entire UI is created to appeal to kids and keep them engaged, adding a custom cursor similar to the ones we generally find in online games would be good, in my opinion. 

If the idea is good, I would like to work on this feature

**Comment by prakhartiwari0** [comment]
@sreevyshnavi02 This is a good idea, I would like to know how it can be done, and if you can implement it 😄

**Comment by sreevyshnavi02** [comment]
Yeah! I can implement it. I have seen that we just require css and vanilla js to create a custom cursor.

**Comment by loki1205** [comment]
Hi can you please assign this to me, I would love to work on this one

**Comment by prakhartiwari0** [comment]
@loki1205 sure, I think @sreevyshnavi02 is inactive. I am unassigning her and assigning you this issue.

**Comment by oshinsaraf** [comment]
Hi can you assign this issue to me, I would like to work on this under GSSOC'23.

**Comment by prakhartiwari0** [comment]
@oshinsaraf I need to first ask from the person who was assigned this issue, they have not completed the PR they made, please read the whole convo here: #94 
If the person doesn't respond, please share your own points there about what can be done in the context of the already happened discussion.

Happy Contributing!
